packages/flame/lib/src/devtools_helpers/dev_tools_service.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flame/game.dart';
+import 'package:flame/src/devtools_helpers/connectors/component_count_connector.dart';
+import 'package:flame/src/devtools_helpers/connectors/debug_mode_connector.dart';
+import 'package:flame/src/devtools_helpers/connectors/game_loop_connector.dart';
+import 'package:flame/src/devtools_helpers/dev_tools_connector.dart';
+
+/// When [DevToolsService] is initialized by the [FlameGame] it will call
+/// the `init` method for all [DevToolsConnector]s so that they can register
+/// service extensions which are the ones that makes it possible for the
+/// devtools extension to communicate with the game.
+///
+/// Do note that if you have multiple games in your app, only the last one
+/// created will be connected to the devtools. If you want to change it to
+/// another game instance you can call [DevToolsService.initWithGame] with
+/// the game instance that you want to observe.
+class DevToolsService {
+  DevToolsService._();
+
+  static final instance = DevToolsService._();
+
+  /// Initializes the service with the given game instance.
+  factory DevToolsService.initWithGame(FlameGame game) {
+    instance.game = game;
+    instance.initGame();
```

Comment:
You should probably check if it is the same game here. Also, you need to call `disposeGame` on all connectors before changing the reference.

Reply:
Good catch! Added this logic at like 3 am last night 😄